### PR TITLE
AWS Account API

### PR DESCRIPTION
**Pull request by lardcanoe**
**Description:**

**Review:**

- [ ] Do the unit tests contain positive functionality tests as well as edge case negative tests?
- [ ] Can we monitor the performance/effectiveness of these changes?
- [ ] Did the code review consider failure modes and edge cases?
- [ ] Did the code review consider refactorings that would make this code more maintainable?
- [ ] Was this change approved by a module owner (if necessary)?

### CODE REVIEW

File: account_api.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+CloudHealth Account API
+=======================
+
+The Account API is used to administer Cloud Provider and Third Party integration accounts.
+
+# AWS Accounts
+
+## Required Fields:
```
Comment (aschepis) [inline account_api.md]: would be so much better if we could point people to generated API doc.

https://github.com/apotonick/roar/issues/139 (not a solution, but ideas.)

we could use the ideas above to generate an [apidoc](http://apidocjs.com/) or [swagger](http://swagger.io/) definition.

other possibilities include:
https://apiblueprint.org
the stuff here: http://www.mattsilverman.com/2013/02/tools-to-generate-beautiful-api-documentation.html

File: account_api.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+* AWS Config
+    * `enabled`: Whether to collect AWS Config files. Off by default.
+    * `bucket`: S3 bucket containing the files
+    * `prefix`: Prefix of files if configured
+    
+* CloudWatch
+    * `enabled`: Whether to collect CloudWatch data. On by default.
+
+## Computed Read-only Fields
```
Comment (aschepis) [inline account_api.md]: computed vs discovered?  what does this mean?  how do customers know what it means?

**Comment by aschepis** [comment]
:+1:

File: account_api.md (hunk 1 above)
Comment (lardcanoe) [inline account_api.md]: Agreed.

I had looked into a lot of those already. Nothing "just works" with what we have currently.

When we get more serious about the API this will get addressed.
